website/docs/r/vapp_vm.html.markdown

```diff
@@ -74,11 +75,12 @@ resource "vcd_vapp_vm" "web2" {
 The following arguments are supported:
 
 * `vapp_name` - (Required) The vApp this VM should belong to.
-* `name` - (Required) A unique name for the vApp
+* `name` - (Required) A unique name for the VM
 * `catalog_name` - (Required) The catalog name in which to find the given vApp Template
 * `template_name` - (Required) The name of the vApp Template to use
-* `memory` - (Optional) The amount of RAM (in MB) to allocate to the vApp
-* `cpus` - (Optional) The number of virtual CPUs to allocate to the vApp
+* `memory` - (Optional) The amount of RAM (in MB) to allocate to the VM
+* `cpus` - (Optional) The number of virtual CPUs to allocate to the VM. Socket count is a result of: virtual logical processors/cores per socket
+* `cpu_cores` - (Optional) The number of cores per socket
```

Comment:
We need to add the version "tag" to this parameter as it's new only to this release, while the resource is not new. Please also think whether there may be other version tags on parameters that we may have missed on old resources.

Reply:
Added